packages/core/src/modules/indy/services/IndyHolderService.ts

```diff
@@ -22,15 +24,19 @@ export class IndyHolderService {
     credentialDefinitions,
     revocationStates = {},
   }: CreateProofOptions) {
-    return this.indy.proverCreateProof(
-      this.indyWallet.walletHandle,
-      proofRequest,
-      requestedCredentials,
-      this.indyWallet.masterSecretId,
-      schemas,
-      credentialDefinitions,
-      revocationStates
-    )
+    try {
+      return await this.indy.proverCreateProof(
+        this.indyWallet.walletHandle,
```

Comment:
`IndyLedgerService` is also using `IndyWallet`, but we name it just as `this.wallet`. I'm quite good with having just `this.wallet`, but if we want to use `this.indyWallet`, we should use it consistently.

Reply:
Maybe even just `this.wallet.handle`? 🤔